crates/forge_main/src/ui.rs

```diff
@@ -207,9 +209,7 @@ impl<F: API> UI<F> {
                                     TRACKER.set_conversation(conversation).await;
                                 }
                             }
-                            tokio::spawn(
-                                TRACKER.dispatch(forge_tracker::EventKind::Error(format!("{error:?}"))),
-                            );
```

Comment:
I think we should not break our existing dashboards. Lets keep this back. Apologies for going back and forth on it.